pd-mapper.c

```diff
@@ -220,6 +237,8 @@ static int pd_enumerate_jsons(struct assoc *json_set)
 	size_t len;
 	size_t n;
 
+	read_fw_path_from_sysfs(fw_sysfs_path, sizeof(fw_sysfs_path));
```

Comment:
The code for this function was taken from related tqftpserv PR, I though it would be easier to review if they are identical... But this can be done, yeah.. I'll think about it

Reply:
The idea is to get rid of complicated "if" below, where we first try this, then try that..?

Reply:
yes